### PR TITLE
build - make cxx_abi=0 default

### PR DESCRIPTION
**Pull request by kanvi-nervana**
Build options: Build with CXX_ABI=0 by default

### CODE REVIEW

**Review by adk9 — CHANGES_REQUESTED** [review]
Why can't one use prebuilt-tensorflow with OV built with _GLIBCXX_USE_CXX11_ABI=0? Perhaps we should default to building everything with _GLIBCXX_USE_CXX11_ABI=0?

**Comment by kanvi-nervana** [comment]
> Why can't one use prebuilt-tensorflow with OV built with _GLIBCXX_USE_CXX11_ABI=0? Perhaps we should default to building everything with _GLIBCXX_USE_CXX11_ABI=0?

You could, you would need to make sure that when you use these 2 options together then you provide the right location of OV built with ABI=0
Fixing the build to use ABI=0 always is a better option.

**Review by adk9 — APPROVED** [review]

File: build_ngtf.py [hunk 1]
```diff
@@ -229,8 +228,9 @@ def main():
     # this is set to 1
```
Comment (bani-intelaipg) [inline build_ngtf.py:228]: This comment should be updated?

**Review by bani-intelaipg — APPROVED** [review]

File: build_ngtf.py (hunk 1 above)
Comment (kanvi-nervana) [reply build_ngtf.py:228]: It is comment in general

**Pull request merged**